deploy/07_deploy_coverage_pool.js

```diff
@@ -2,12 +2,22 @@ module.exports = async ({ getNamedAccounts, deployments }) => {
   const { deploy } = deployments
   const { deployer } = await getNamedAccounts()
   const AssetPool = await deployments.get("AssetPool")
+  const UnderwriterToken = await deployments.get("UnderwriterToken")
 
-  await deploy("CoveragePool", {
+  const CoveragePool = await deploy("CoveragePool", {
     from: deployer,
     args: [AssetPool.address],
     log: true,
   })
+
+  const assetPool = await ethers.getContractAt("AssetPool", AssetPool.address)
+  const underwriterToken = await ethers.getContractAt(
```

Comment:
We can simplify it to:
```js
  const assetPool = await ethers.getContract("AssetPool")
  const underwriterToken = await ethers.getContract("UnderwriterToken")
```

But this will also require some configuration. According to [installation instructions](https://github.com/wighawag/hardhat-deploy#installation) we need to:
1. Run `npm install --save-dev  @nomiclabs/hardhat-ethers@npm:hardhat-deploy-ethers ethers`.
2. Add `require("@nomiclabs/hardhat-ethers")` in `hardhat.config.js`.

It's not a big deal but it may be worth doing it now to establish a better pattern for the future scripts enhancement.

Reply:
I've played with ^, but unfortunately, tests fail after installation. I think there is too much magic around dependencies and the `ethers.getSigner` function from the `hardhat-deploy-ethers` is not compatible with the same function from the ` @nomiclabs/hardhat-ethers`.

Reply:
For each invocation `ethers.getSigner` in tests with updated hardhat config I received `Error: need to specify address`.

Reply:
Thanks for trying! The current solution is super fine, if we can't improve it easily I don't want to block this PR. We will handle it in a follow-up.

Reply:
I managed to get it done in https://github.com/keep-network/coverage-pools/pull/143 :tada: